charts/kubernetes-zfs-provisioner/templates/storageclass.yaml

```
@@ -19,5 +19,6 @@ parameters:
   type: {{ .type | default "nfs" }}
   node: {{ .node | default "''" }}
   shareProperties: {{ .shareProperties | default "''" }}
+  reserveSpace: {{ .reserveSpace | default "true" }}
```

The Helm run still fails, because when rendered, `true` becomes a bool in YAML, e.g. `reserveSpace: true`, whereas the expectation is a string. To be extra sure, we should quote it.
```suggestion
  reserveSpace: {{ .reserveSpace | default "'true'" }}
```

---

I'm not sure my first suggestion works, maybe this works
```suggestion
  reserveSpace: {{ .reserveSpace | default (quote true) }}
```
or `{{ .reserveSpace | default (true | quote) }}`

---

I can't explain why, but this still didn't work. I've found another solution though, I've also added chart unit tests for this property.